src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -136,6 +136,22 @@ pscanalpha.stricttransportsecurity.max.age.name=Strict-Transport-Security Disabl
 pscanalpha.stricttransportsecurity.max.age.desc=A HTTP Strict Transport Security (HSTS) header was found, but it contains the directive max-age=0 which disables the control and instructs browsers to reset any previous HSTS related settings. See RFC 6797 for further details.\nHTTP Strict Transport Security (HSTS) is a web security policy mechanism whereby a web server declares that complying user agents (such as a web browser) are to interact with it using only secure HTTPS  connections (i.e. HTTP layered over TLS/SSL).
 pscanalpha.stricttransportsecurity.max.age.refs=http://tools.ietf.org/html/rfc6797#section-6.2
 pscanalpha.stricttransportsecurity.max.age.soln=Review the configuration of this control. Ensure that your web server, application server, load balancer, etc. is configured to set Strict-Transport-Security with an appropriate max-age value.
+pscanalpha.stricttransportsecurity.compliance.mutiple.header.name=Strict-Transport-Security Multiple Header Entries (Non-compliant with Spec)
+pscanalpha.stricttransportsecurity.compliance.mutiple.header.desc=HTTP Strict Transport Security (HSTS) headers were found, a response with multiple HSTS header entries is not compliant with the specification (RFC 6797) and only the first HSTS header will be processed others will be ignored by client browsers or the HSTS policy may be incorrectly applied.\nHTTP Strict Transport Security (HSTS) is a web security policy mechanism whereby a web server declares that complying user agents (such as a web browser) are to interact with it using only secure HTTPS  connections (i.e. HTTP layered over TLS/SSL).
```

Comment:
- The text "client browsers" could be replaced with "user agents" to be consistent with other messages.
 - There's an extra space in "HTTPS connections", it's also in other messages, including an old one.